server/service/integration_mdm_test.go

```diff
@@ -6668,8 +6674,18 @@ func (s *integrationMDMTestSuite) TestValidRequestSecurityTokenRequestWithDevice
 			"host_display_name": "DESKTOP-0C89RC0"
 		 }`,
 		0)
+
+	expectedDeviceID := "AB157C3A18778F4FB21E2739066C1F27" // TODO: make the hard-coded deviceID in `s.newSecurityTokenMsg` configurable
+
+	// Checking if the host uuid was set on mdm windows enrollments
+	d, err := s.ds.MDMWindowsGetEnrolledDeviceWithDeviceID(context.Background(), expectedDeviceID)
+	require.NoError(t, err)
+	require.NotEmpty(t, d.HostUUID)
+	require.Equal(t, windowsHost.UUID, d.HostUUID)
 }
 
+// TODO: Do we need integration tests for WindowsMDMAutomaticEnrollmentType flows?
```

Comment:
I think we do, I'd say let's do them when the commands PR that introduces the mock hosts lands.